integration-tests/src/test/java/com/datastax/oss/driver/api/core/ConnectIT.java

```diff
@@ -87,8 +87,9 @@ public void should_wait_for_contact_points_if_reconnection_enabled() throws Exce
     DriverConfigLoader loader =
         SessionUtils.configLoaderBuilder()
             .withBoolean(DefaultDriverOption.RECONNECT_ON_INIT, true)
-            .withClass(
```

Comment:
Why remove `withClass`?

Reply:
It has very little value, I thought it wasn't worth spreading the API surface (if we add it, why not also handle enum values, etc).

Reply:
I wonder if such methods, although of little value for developers, aren't appreciated by users. I would advocate for us supporting classes and enum values here (specially given that TypeSafe has built-in support for enum values). This is the typical type of nice-to-haves that makes the user experience more streamlined.

Reply:
I've readded withClass.